spec/unit/shoulda/matchers/active_model/validate_numericality_of_matcher_spec.rb

```diff
@@ -200,17 +220,24 @@
         expect(validating_numericality(even: true, greater_than: 18))
           .to matcher.even.is_greater_than(18)
       end
+
       it do
         expect(validating_numericality(odd: true, less_than_or_equal_to: 99))
           .to matcher.odd.is_less_than_or_equal_to(99)
       end
 
       it do
```

Comment:
I think you can remove this test. I feel confident that `greater_than` is tested along with `strict` in comparison_matcher_spec.